addon/system/uuid.js

```diff
@@ -26,8 +26,8 @@ let random = (function () {
       }
 
       numbers[i] = r >>> ((i & 0x03) << 3) & 0xFF;
-      return numbers;
     }
+    return numbers;
```

Comment:
This was causing the build to fail. It's not related to the work at-hand, so if you want, I can put it in a separate PR.

Reply:
This is fine here :)